server/routes/animeList.js

```diff
@@ -4,3 +4,20 @@ const router = express.Router();
 import * as dotenv from 'dotenv';
 dotenv.config();
 
+
+router.get('/', async (req, res) => {
+    const sub = req.query.sub;
+    const userSub = `SELECT id FROM users WHERE sub =$1`;
+    const subInfo = await db.query(userSub, [sub])
+    console.log(subInfo, "Finds respective user id from given sub");
+    const user_id = subInfo[0].id;
+  
+    try {
+      const itemList = await db.any(`SELECT * FROM anime WHERE user_id=$1`, [user_id]);
```

Comment:
WORKING QUERY ON POSTICO :
SELECT a.*, u.complete
FROM anime as a
JOIN user_anime_list as u
ON a.mal_id = u.anime_id
WHERE u.user_id=5;

Goal: able to grab the anime data to call fetch request in the front end
a complete column was added default false (this can be expanded upon later if i have time)